src/components/Model3DViewer/stories/full.md

```diff
@@ -74,7 +74,8 @@ function ExampleComponent(props) {
 | `assetId`                  | id of asset to highlight                                            | `number`                                                                            |         |
 | `boundingBox`              | bounding box object, that describes dimension of viewed asset nodes | `THREE.Box3`                                                                        |         |
 | `cache`                    | object for caching 3D viewers instances                             | `{ [name:string]: any }`                                                            |   {}    |
-| `useDefaultCameraPosition` | use default camera position                                         | `boolean`                                                                           | true    |
+| `disableKeyboardNavigation`| disable/enable keyboard navigation in viewer (disabled by default)  | `boolean`                                                                           |  true   |
```

Comment:
Maybe it's better to make prop `enableKeyboardNavigation` instead since `disableKeyboardNavigation` is true by default?

Reply:
make sense